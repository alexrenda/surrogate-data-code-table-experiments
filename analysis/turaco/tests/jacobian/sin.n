fun (x) {
  x = x * 10;
  x = sin(x);
  return x;
}