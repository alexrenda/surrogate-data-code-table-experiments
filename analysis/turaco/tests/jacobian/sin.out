 110132.32920103322 10.0
